mypy/stubgen.py

```diff
@@ -647,11 +647,14 @@ def process_name_expr_decorator(self, expr: NameExpr, context: Decorator) -> boo
                                                           'asyncio.coroutines',
                                                           'types'):
             self.add_coroutine_decorator(context.func, name, name)
-        elif any(self.refers_to_fullname(name, target)
-                 for target in ('abc.abstractmethod', 'abc.abstractproperty')):
+        elif self.refers_to_fullname(name, 'abc.abstractmethod'):
             self.add_decorator(name)
             self.import_tracker.require_name(name)
             is_abstract = True
+        elif self.refers_to_fullname(name, 'abc.abstractproperty'):
+            self.add_decorator('property')
+            self.add_decorator('abc.abstractmethod')
+            is_abstract = True
```

Comment:
I'd like to be more specific about here. (The previous commit has incorrect tests)
I intentionally did not require_name in import_tracker. This is because `is_abstract` will be set and `abc` will be imported, so directly add `abc.abstractmethod` would work. And as shown in testcase: `testAbstractProperty3_semanal`, if the imported abstractproperty is aliased, to properly split would have to modify code in import_tracker, which IMO is not necessary since using `abc.abstractmethod` already achieves purposes